runelite-client/src/test/java/net/runelite/client/plugins/timers/TimersPluginTest.java

```diff
@@ -204,28 +204,28 @@ public void testTransparentChatboxTbRemoved()
 	public void testMageArena2TbFull()
 	{
 		when(timersConfig.showTeleblock()).thenReturn(true);
-		ChatMessage chatMessage = new ChatMessage(null, ChatMessageType.SPAM, "", "<col=c356ef>A Tele Block spell has been cast on you. It will expire in 2 minutes.</col>", "", 0);
-		timersPlugin.onChatMessage(chatMessage);
+		when(client.getVarbitValue(Varbits.TELEBLOCK)).thenReturn(300);
```

Comment:
There's no need to keep all the teleblock tests now that it doesn't have to parse numerous text variants. You can keep a minimal number to test the various varbit value tiers or just remove them entirely.